README.md

```diff
@@ -74,38 +50,38 @@ success.
 For more information look here:
 [Strategic Initiatives](https://github.com/planetarysoftware/TSC/blob/master/Strategic-Initiatives.md)
 
-## Top-Level WGs and TLPs
-
-* [Working Groups](Working-Groups.md)
-* Top-Level Projects
 
 ## Policy Change Proposal Process
 
 The Planetary Software TSC is chartered to oversee the technical governance of all Top
 Level Projects and Working Groups under the Planetary Software Organization. The TSC
-establishes the default governance, conduct, and licensing policies for all Top
-Level Projects. Top Level Projects and Working Groups have broad powers of
+establishes the default governance, conduct, and licensing policies for all Top-Level 
+Projects. Top-Level Projects and Working Groups have broad powers of
 self-governance.
 
 To propose a change or addition to policies or processes that are intended to
-cover all Top Level Projects and Working Groups in the foundation, a PR should
-be opened in the `planetarysoftware/TSC` repository.
-
-The pull request can be labeled `tsc-agenda` to request that it be put on the
-agenda for the next TSC meeting.
+cover all Top-Level Projects and Working Groups in the Planetary Software Organization,
+a PR should be opened in the `planetarysoftware/TSC` repository.
 
 In some cases, existing individual groups have the right to refuse changes to
 their charters. The TSC cannot mandate existing working groups alter their
 charters. If such a situation arises, the TSC may decide to revoke the group's
 charter.
 
 ## Merging Pull Requests in the TSC Repo
-The submitter of a pull request (PR) is responsible for making changes to the text of the PR.  Any changes to the PR can be suggested by others in the PR thread, but changes to the PR should be made by the PR author.  In order to merge a PR, it must satisfy both of these two conditions:
+In general, the submitter of a pull request (PR) is responsible for
+making changes to the text of the PR.  Any changes to the PR can
+be suggested by others in the PR thread, but changes to the PR
+should be made by the PR author.  In order to merge a PR, it must
+satisfy both of these two conditions:
 
   1. have been open for 24 hours
   2. have two approvals
 
-Exceptions to the above are minor typo fixes or cosmetic changes that don't alter the meaning of a document.  Those edits can be made via a PR which only requires a single approval (not two, and not open for 24 h) to be merged.
+Exceptions to the above are minor typo fixes or cosmetic changes
+that don't alter the meaning of a document.  Those edits can be
+made via a PR which only requires a single approval (not two, and
+not open for 24 h) to be merged.
 
-[TSC Charter]: https://github.com/planetarysoftware/TSC/blob/master/TSC-Charter.md
-[Project Lifecycle.md]: ./Project-Lifecycle.md
+Some kinds of PRs have different rules (those to change the TSC Charter, 
+applications for TLPs and WGs, etc.) and will be detailed in those sections.
```

Comment:
Maybe change "in those sections" to "in the respective governing documents" or something like that, reflecting that it's not describe in another section of this document?

Reply:
Good point.  I'll push a fix for this.